.binder/postBuild

```diff
@@ -3,17 +3,10 @@ set -eux
 
 pushd js
 jlpm --ignore-optional
-jlpm build:bundles
-jlpm build:all
 popd
 
-python -m pip install -vv -e . --no-deps --ignore-installed
-python -m pip check
-
-# install labextension
-pushd js
-jupyter labextension develop . --overwrite
-popd
+python setup.py sdist
+python -m pip install -v dist/pythreejs*.tar.gz --no-deps --ignore-installed
```

Comment:
Does this not cause a new build when installing from the sdist? I might be more optimal to install via wheel?

Reply:
it probably does... i'm more interested in _robust_ than _optimal_, and frankly care more about what makes it to the (intermediate) `.tar.gz` than the `.whl`... but faster is of course better. will try a few more things...

Reply:
`--no-build-isolation` saves a some stuff (e.g recompiling `pyrsistent` and `pandocfilters`).

Unfortunately, the way `--skip-npm` is passed down, if you have node, it's going to rebuild... perhaps optionally checking a `PYTHREEJS_SKIP_NPM` would be more robust. Or if the `stale` check worked...

Reply:
To be clear, my suggestion was to use `python setup.py bdist_wheel` and then pip install using the wheel. That should do what you want while still avoiding rebuilds, right ?

Reply:
(if you are saying this is meant to be a test of the sdist install, that is probably different).